src/core/Akka/Actor/SupervisorStrategy.cs

```diff
@@ -80,17 +80,10 @@ public bool HandleFailure(ActorCell actorCell, Exception cause, ChildRestartStat
         /// </summary>
         /// <param name="exception">The exception.</param>
         /// <returns>Directive.</returns>
-        public static Directive DefaultDecider(Exception exception)
-        {
-            if (exception is ActorInitializationException)
-                return Directive.Stop;
-            if (exception is ActorKilledException)
-                return Directive.Stop;
-            if (exception is DeathPactException)
-                return Directive.Stop;
-
-            return Directive.Restart;
-        }
+        public static IDecider DefaultDecider = Decider.From(Directive.Restart,
+            Directive.Stop.When<ActorInitializationException>(),
+            Directive.Stop.When<ActorKilledException>(),
+            Directive.Stop.When<DeathPactException>());
```

Comment:
The old default decider have been replaced with a deployable decider with the exact same behavior.
I consider this a better default value as supervisorstrategies and pool routers are deployable when created using their small constructors.